lib/awful/client.lua

```diff
@@ -634,6 +634,42 @@ function client.object.is_fixed(c)
     return false
 end
 
+--- Is the client immobilized horizontally, i.e. fullscreen or maximized?
```

Comment:
I'd change this to `Does the client have a fixed horizontal position and width, i.e. is it fullscreen, maximized, or horizontally maximized?`

(I guess: same for `is_immobilized_horizontal`).

Reply:
Moved to the long desc.